CMakeLists.txt

```diff
@@ -356,7 +356,7 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
 
   # The machete kernels only work on hopper and require CUDA 12.0 or later.
   # Only build Machete kernels if we are building for something compatible with sm90a
-  cuda_archs_loose_intersection(MACHETE_ARCHS "9.0a" "${CUDA_ARCHS}")
+  cuda_archs_loose_intersection(MACHETE_ARCHS "9.0a;10.0;12.0" "${CUDA_ARCHS}")
   if(${CMAKE_CUDA_COMPILER_VERSION} VERSION_GREATER 12.0 AND MACHETE_ARCHS)
```

Comment:
We'll need to modify CMakeLists to only append 10.0 and 12.0 when CMAKE_CUDA_COMPILER_VERSION is greater than 12.7 and 12.8 respectively

(Here and in other places)

Reply:
i will do, if anyone wants edit also, feel free

Reply:
Well, I'm changing it